nncf/config_schema.py

```diff
@@ -174,31 +174,56 @@ def with_attributes(schema: Dict, **kwargs) -> Dict:
     "additionalProperties": False,
 }
 
+BASIC_RANGE_INIT_CONFIG_PROPERTIES = {
+    "type": "object",
+    "properties": {
+        "num_init_steps": with_attributes(_NUMBER,
+                                            description="Number of batches from the training dataset to "
+                                                        "consume as sample model inputs for purposes of "
+                                                        "setting initial minimum and maximum quantization "
+                                                        "ranges"),
+        "type": with_attributes(_STRING, description="Type of the initializer - determines which "
+                                                        "statistics gathered during initialization will be "
+                                                        "used to initialize the quantization ranges"),
+        "min_percentile": with_attributes(_NUMBER,
+                                            description="For 'percentile' type - specify the percentile of "
+                                                        "input value histograms to be set as the initial "
+                                                        "value for minimum quantizer input"),
+        "max_percentile": with_attributes(_NUMBER,
+                                            description="For 'percentile' type - specify the percentile of "
+                                                        "input value histograms to be set as the initial "
+                                                        "value for maximum quantizer input"),
+    },
+    "additionalProperties": False,
+}
+EXTENDED_RANGE_INIT_CONFIG_PROPERTIES = {
```

Comment:
This is only "extended" with a per-layer specification functionality. Should call it `PER_LAYER_RANGE_INIT_CONFIG_PROPERTIES`, perhaps.

Reply:
Done.